docs/source/opentelemetry.mdx

```diff
@@ -1,12 +1,14 @@
 ---
-title: OpenTelemetry in Apollo Server
+title: OpenTelemetry in Apollo Federation
 sidebar_title: OpenTelemetry
 ---
 
 import {
   ExpansionPanel,
 } from 'gatsby-theme-apollo-docs/src/components/expansion-panel';
 
+> 📣 **Open Telemetry demo:** In addition to completing this tutorial, you can [clone our Federation demo on GitHub](https://github.com/apollographql/supergraph-demo-fed2). The demo is distinct from the project you create in this tutorial, and it helps demonstrate the use of [Open Telemetry in Federation](https://github.com/apollographql/supergraph-demo-fed2#tracing-with-open-telemetry).
```

Comment:
I love the fully-functional end-to-end demo, and I know it's called "Demo", butt I find it a bit incongruent to send someone off to a completely "distinct" experience.   I'm worried that it itself will become a pattern, particularly because it has other moving pieces which are not supported, including the underlying [`supergraph-demo-opentelemetry`](https://npm.im/supergraph-demo-opentelemetry) package which — while great — is still a maintenance concern and something we'll need to think about and potentially support.  We can commit to updating these docs right now and going forward, but docs were what we agreed to for this.

Thoughts?  Maybe there's another treatment we can give this? I think the happy path should ideally be outlined below, but maybe there's another way to position it?